dev-docs/modules/arcspanRtdProvider.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: page_v2
+title: ArcSpan Contextual APP 
+display_name: ArcSpan Contextual APP RTD Provider
+description: Surfaces contextual classification signals in the bid request to enhance the addressability of ad impressions and increase buyer demand
+page_type: module
+module_type: rtd
+module_code : arcspanRtdProvider
+enable_download : true
+vendor_specific: true
+sidebarType : 1
+---
+
+# ArcSpan Contextual APP RTD Provider
+{:.no_toc}
+
+* TOC
+{:toc}
+
+## Prebid Config for ArcSpan RTD Module
+
+This module attaches contextual classification signals to the site object in the bid request, with the goal of enhancing the addressability of ad impressions and increasing buyer demand.
+
+### Usage
+
+Compile the ArcSpan RTD Module into your Prebid build:
+
+```
+gulp build --modules=rtdModule,arcspanRtdProvider
+```
+
+{: .alert.alert-warning :}
+Note that the global RTD module, `rtdModule`, is a prerequisite of the ArcSpan RTD Module.
+
+You then need to enable the ArcSpan RTD Module in your Prebid configuration, using the format below.
+
+{: .alert.alert-warning :}
+Please replace the `silo` parameter value with the one provided by your ArcSpan representative.
```

Comment:
Please indicate the silo param actually determines which external js will load, that this js is or is not locked to a version and that prebid has not warrantied nor reviewed this external js.